packages/cli/cloud/src/create-project/action.ts

```diff
@@ -0,0 +1,61 @@
+import inquirer from 'inquirer';
+import { AxiosError } from 'axios';
+import { defaults } from 'lodash/fp';
+import type { CLIContext, ProjectAnswers, ProjectInput } from '../types';
+import { tokenServiceFactory, cloudApiFactory, local } from '../services';
+
+function handleError(ctx: CLIContext, error: Error) {
+  const tokenService = tokenServiceFactory(ctx);
+  const { logger } = ctx;
+
+  logger.debug(JSON.stringify(error));
+  if (error instanceof AxiosError) {
+    switch (error.response?.status) {
+      case 401:
+        logger.error('Your session has expired. Please log in again.');
+        tokenService.eraseToken();
+        return;
+      case 403:
+        logger.error(
+          error.response.data ||
+            'You do not have permission to create a project. Please contact support for assistance.'
+        );
+        return;
+      case 400:
+        logger.error('Invalid input. Please check your inputs and try again.');
+        return;
+      default:
+        break;
+    }
+  }
+  logger.error(
+    'We encountered an issue while creating your project. Please try again in a moment. If the problem persists, contact support for assistance.'
```

Comment:
Should you propose using the `debug` flag to find more info?

Reply:
I don't believe we want users to use debug flag without being invited to. Advanced users would benefit from it, but it would add a lot of extra information not needed for a regular user. I think it will be more convenient if the support ask the user to use it and send the logs to us in order to fix the issue. 🤔